x/leverage/handler.go

```diff
@@ -1,14 +1,13 @@
-// nolint: gocritic
-package umee
+package leverage
```

Comment:
I think this file came from the Starport scaffolding. We actually don't need it at all -- we can delete it.

The way it'll work is, you define the message types in `tx.proto` along with the message server (see above comment for x/bank example). Then this will generate a `msg_server.go` file that will implement the handler scaffolding.

Reply:
If we take it out, the following in `x/leverage/module.go` has a problem:
```
// Route returns the capability module's message routing key.
func (am AppModule) Route() sdk.Route {
	return sdk.NewRoute(types.RouterKey, NewHandler(am.keeper))
}
```
where `NewHandler` is what's been deleted.

What do we replace the function body with?

Reply:
Note: When adding a msg service to tx.proto 
```
// Msg defines the leverage Msg service.
service Msg {
  // Lend defines a method for lending coins to the capital facility.
  rpc Lend(MsgLendAsset) returns (MsgLendAssetResponse);

  // Withdraw defines a method for withdrawing previously lent coins from the capital facility.
  rpc Withdraw(MsgWithdrawAsset) returns (MsgWithdrawAssetResponse);
}
```
I get this warning:
```
umee % make proto-gen
Generating Protobuf files
W0923 23:57:50.013284     107 services.go:38] No HttpRule found for method: Msg.Lend
W0923 23:57:50.013413     107 services.go:38] No HttpRule found for method: Msg.Withdraw
```
They may be related to msg_server.go not being generated. The service change, but not the handler file deletion, is pushed to my branch right now, so you can reproduce it.

Reply:
Yup, so the `No HttpRule` warning we can ignore. The SDK also generates these warnings ;-)